TouchEffect.Droid/PlatformTouchEff.cs

```diff
@@ -58,19 +58,75 @@ protected override void OnDetached()
             }
         }
 
+        bool _inrange;
+        bool _pressed;
+        bool _leftrange = false;
```

Comment:
no need to set "false" as it's default value for bool.
And what is the purpose of this flag? we have _inRange (this is the same, no?)

Reply:
It's indeed pointless, apologies (as was `_pressed` in fact)